lib/api/kuzzle.js

```diff
@@ -342,7 +341,9 @@ function dumpAndExit(kuzzle, suffix) {
     runShutdown(kuzzle);
   }
 
-  kuzzle.janitor.dump(suffix).finally(() => runShutdown(kuzzle));
+  kuzzle.janitor.dump(suffix)
+    .catch(() => {}) // do nothing, prevents unhandled rejections
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this function is empty or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1186)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AEmptyBlock)